tests/p2p_tests/sync/message_storm.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+###############################################################
+# -p <producing nodes count>
+# -n <total nodes>
+# -s <topology>
+# -d <delay between nodes startup>
+###############################################################
+
+pnodes=21
+topo=mesh
+delay=5
+
+args=`getopt p:n:s:d: $*`
+if [ $? == 0 ]; then
+
+    set -- $args
+    for i; do
+        case "$i"
+        in
+            -p) pnodes=$2;
+                shift; shift;;
+            -n) total_nodes=$2;
+                shift; shift;;
+            -d) delay=$2;
+                shift; shift;;
+            -s) topo="$2";
+                shift; shift;;
+            --) shift;
+                break;;
+        esac
+    done
+else
+    echo "huh we got err $?"
+    if [ -n "$1" ]; then
+        pnodes=$1
+        if [ -n "$2" ]; then
+            topo=$2
+            if [ -n "$3" ]; then
+		total_nodes=$3
+            fi
+        fi
+    fi
+fi
+
+total_nodes="${total_nodes:-`echo $pnodes`}"
+
+rm -rf tn_data_*
+if [ "$delay" == 0 ]; then
+    programs/launcher/launcher -p $pnodes -n $total_nodes -s $topo
+else
+    programs/launcher/launcher -p $pnodes -n $total_nodes -s $topo -d $delay --eosd "--log-level-net-plugin debug"
+fi
+
+sleep 7
+echo "start" > test.out
+port=8888
+endport=`expr $port + $total_nodes`
+echo endport = $endport
+while [ $port  -ne $endport ]; do
+    programs/eosc/eosc --port $port get block 1 >> test.out 2>&1;
+    port=`expr $port + 1`
+done
+
+grep 'producer"' test.out | tee summary | sort -u -k2 | tee unique
+prodsfound=`wc -l < unique`
+lines=`wc -l < summary`
+if [ $lines -eq $total_nodes -a $prodsfound -eq 1 ]; then
+    echo all synced
+#    programs/launcher/launcher -k 15
+#    exit
```

Comment:
Did you mean to leave this commented out?

Reply:
yes. To get enough run time for properly detecting the message storm.